homeassistant/components/tuya/select.py

```diff
@@ -52,6 +52,16 @@
             icon="mdi:coffee",
         ),
     ),
+    #Motion Sensor
+    #https://developer.tuya.com/en/docs/iot/s?id=K9gf48cz3utjt
+    "pir": (
```

Comment:
Sorry for wasting Your time, I was trying to begin to contribute but looks like I should go back to the drawing board! 
I was trying to add support for the pir sensors. There was quite a few tickets in the old tuyav2 that were trying to get it working as well as #68706 and #63356

Reply:
That is OK, but I'm not fully understanding the change made in this PR, hence me asking if you can elaborate on this change.

Reply:
You can reject this change. I thought I need to add PIR to this file has it was missing here. Looking through and trying to understand how this component works I see PIR is already added to the binary_sensor.py. When I check the API in postman it shows value = pir for only a few seconds before going offline to which is shows value = none.
{
    "result": [
        {
            "code": "pir",
            "value": "pir"
        },
        {
            "code": "battery_percentage",
            "value": 100
        }
    ],
    "success": true,
    "t": 1657849874540,
    "tid": "9c1e18cb12e011edbf867eeb5c0b2a02"
}
In Home Assistant it just always shows as unavailable. Its as if because its 'offline' the motion detections never make it through or automatically make it disabled. 

Postman shows this for the device details.

![image](https://user-images.githubusercontent.com/82223761/179131306-22f2ab4c-3123-4f8c-a34d-38179b3ca4a9.png)

    "result": {
        "active_time": 1655738863,
        "biz_type": 18,
        "category": "pir",
        "create_time": 1655738863,
        "icon": "smart/icon/ay1536534747810ekKBj/1565841334dae0a2029d5.png",
        "id": "bfaafef50c0c067fb51ab7",
        "ip": "",
        "model": "YIDONG",
        "name": "Motion Sensor PIRB Office",
        "online": false,
        "owner_id": "3335541",
        "product_id": "6xcrb0jp",
        "product_name": "Motion Sensor",
        "status": [
            {
                "code": "pir",
                "value": "none"
            },
            {
                "code": "battery_percentage",
                "value": 100
            }
        ],

I really hope I can be able to assist in adding new devices and attributes!

Reply:
> You can reject this change.

Ok 👍 

> In Home Assistant it just always shows as unavailable. Its as if because its 'offline' the motion detections never make it through or automatically make it disabled.

Interesting 👍  Could you please rais an issue in the issue tracker, adding a device diagnostic dumps downloaded from Home Assistant in both `pir` and `none` state values of the `pir` data point?

../Frenck